packages/backend/src/modules/user/user.model.ts

```diff
@@ -0,0 +1,11 @@
+import mongoose from 'mongoose'
+
+const userSchema = new mongoose.Schema({
```

Comment:
What was wrong in older schema?

`import mongoose from 'mongoose'
import { Role } from './Role'

// const { ObjectId } = mongoose.Types

const userSchema = new mongoose.Schema({
  username: String,
  googleId: String,
  avatarUrl: String,
  email: String,
  dashboards: [
    {
      imageCoverId: String,
      title: String,
      role: {
        type: String,
        enum: Role,
        default: Role.NO_ACCESS
      },
      users: [
        {
          avatarId: Number
        }
      ]
    }
  ]
})

const User = mongoose.model('User', userSchema)
export default User`

Reply:
https://github.com/kacper-cyra/camp-trello-clone/pull/112#discussion_r873225690